header.php

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8" />
+      <!-- header.php - Global header for use on each page of the web app
+      CSC450 - Computer Science Capstone
+      Group 1:
+      Elise Frigoli
+      Nolan Harre
+      Joshua Sibert
+      Lor Xiong
+      Written:     10/03/21
+      Revisions:
+      -->
+
+      <!-- Boostrap CSS -->
+      <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+      <!-- Custom CSS -->
+      <link rel="stylesheet" type="text/css" href="style.css">
+      <!-- Google Fonts -->
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;400;700;800&display=swap" rel="stylesheet">
+   </head>
```

Comment:
not sure if this really matters or not, but it seems that the closing tag here on lines 24, 33, 34, 55, 77, and 78 are getting flagged as "errors". Probably still works fine, but could be deleted so it removes the error flag.